cmake/FindZeroMQ.cmake

```diff
@@ -0,0 +1,52 @@
+#[=======================================================================[.rst:
+FindZeroMQ
+-----------
+
+Finds ZeroMQ. Does not (currently) support version matching.
+
+IMPORTED Targets
+^^^^^^^^^^^^^^^^
+
+The following :prop_tgt:`IMPORTED` targets may be defined:
+
+``ZeroMQ::libzmq``
+  ZeroMQ library.
+
+Result variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables in your project:
+
+``ZeroMQ_FOUND``
+  true if ZeroMQ headers and library were found
+``ZeroMQ_INCLUDE_DIR``
+  the directory containing ZeroMQ headers
+``ZeroMQ_LIBRARY``
+  ZeroMQ library to be linked
+
+TODO
+^^^^
+
+* Version matching
```

Comment:
Same here.

Reply:
I'll probably do this when I move the find module into the irods repository.